src/pages/Services.jsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function Services () {
   return (
-    <h1 className="text-4xl text-center">This is the Services page 👋</h1>
+    <div className="pt-16 md:py-8">
+      <div className="pl-64">
+        <div className="pr-64">
+          <h1 className="text-4xl text-center">Services</h1>
+          <br />
+          {/* <h1 className="text-4xl text-center" style={{ fontFamily: 'Lato' }}>Coming soon!</h1> */}
```

Comment:
nitpick(nb): Why do we have the commented out code here? If there's no pressing reason to keep it, then remove for tidiness.